app/helpers/and.js

```diff
@@ -0,0 +1,4 @@
+import Ember from 'ember';
+import { andHelper } from 'ember-truth-helpers/helpers/and';
+
+export default Ember.Helper.helper(andHelper);
```

Comment:
That means that prior to 1.13 application will receive helpers from initialize step, while for 1.13 all of them will be auto-discovered.

Taking this separation of concerns into account I propose we use `Ember.Helper` in `app/helpers/*` without prior check for it.

If it is not reasonable, I will use a few days to see how I can write tests for < 1.13

Thanks for help.